examples/collision_free_trajectory.py

```diff
@@ -49,30 +51,34 @@
 with_base = not args.without_base
 av_start = np.array([0.564, 0.35, -0.74, -0.7, -0.7, -0.17, -0.63])
 if with_base:
-    av_start = np.hstack([av_start, [0, 0, 0]])
+    # base pose is sepcified by [x, y, theta]
+    base_pose_start = [-0.5, 0.8, 0]
+    av_start = np.hstack([av_start, base_pose_start])
 
 # solve inverse kinematics to obtain av_goal
 joint_angles = np.deg2rad([-60, 74, -70, -120, -20, -30, 180])
 set_robot_config(robot_model, joint_list, joint_angles)
+target_coords = skrobot.coordinates.Coordinates([0.8, -0.6, 0.8], [0, 0, 0])
 
 rarm_end_coords = skrobot.coordinates.CascadedCoords(
     parent=robot_model.r_gripper_tool_frame,
     name='rarm_end_coords')
 robot_model.inverse_kinematics(
-    target_coords=skrobot.coordinates.Coordinates([0.8, -0.6, 0.8], [0, 0, 0]),
+    target_coords=target_coords,
     link_list=link_list,
     move_target=robot_model.rarm_end_coords, rotation_axis=True)
 av_goal = get_robot_config(robot_model, joint_list, with_base=with_base)
 
 # collision checker setup
-sscc = SweptSphereSdfCollisionChecker(lambda X: box.sdf(X), robot_model)
+sscc = SweptSphereSdfCollisionChecker(box.sdf, robot_model)
 for link in coll_link_list:
     sscc.add_collision_link(link)
 
 # motion planning
 ts = time.time()
+n_waypoint = 10
```

Comment:
It's good. How about adding argparse's argument?

Reply:
Fixed in https://github.com/iory/scikit-robot/pull/218/commits/71f0f5b43ed6fa686c69c7e01278249a7fa5cad8